sqlite.go

```diff
@@ -717,6 +717,18 @@ func BusyTimeout(sqlconn SQLConn, d time.Duration) error {
 	})
 }
 
+// TxnState calls sqlite3_txn_state on the underlying connection.
+func TxnState(sqlconn SQLConn, schema string) (state sqliteh.Txn, err error) {
```

Comment:
if you rename the type, then you can drop the named result parameters.

the func name would need to change, though.... `ConnTxnState` ?